src/integration-test/charles-client.ts

```diff
@@ -300,64 +299,40 @@ export default class CharlesClient {
     return repoUrl.replace(gitserver, gitServerWithCredentials);
   }
 
-  /**
-   * LOWLEVEL
-   */
-
-  private async fetchJson<T>(path: string, options?: RequestInit, requiredStatus = 200) {
-    const response = await this.fetch(`${this.url}${path}`, options);
-    return getResponseJson<T>(response, requiredStatus);
+  public toDto() {
```

Comment:
it's stands for Data Transfer Object, i.e a serializable version. Used for caching.